CollapseLauncher/XAMLs/MainApp/MainPage.xaml.cs

```diff
@@ -208,14 +199,18 @@ private void UpdateBindingsEvent(object sender, EventArgs e)
             UpdateLayout();
 
             // Find the last selected category/title and region
-            string LastCategory = CurrentConfigV2GameCategory;
-            string LastRegion = CurrentConfigV2GameRegion;
-            int indexOfCategory = ConfigV2GameCategory.IndexOf(LastCategory);
-            int indexOfRegion = ConfigV2GameRegions.IndexOf(LastRegion);
+            string lastName = LauncherMetadataHelper.CurrentMetadataConfigGameName;
+            string lastRegion = LauncherMetadataHelper.CurrentMetadataConfigGameRegion;
+
+            List<string>? gameNameCollection = LauncherMetadataHelper.GetGameNameCollection();
```

Comment:
add `#nullable`

Reply:
We will add the define for this one later since it's a big chunk of codes in a file